tests/integration/event_filters_test.go

```diff
@@ -1700,6 +1700,336 @@ func Test_EventFilters(t *testing.T) {
+						expectEvent(anyHost, "cp", testutils.CPUForTests, anyPID, 0, events.SecurityFileOpen, orPolNames("sfo-mw-pol-1"), orPolIDs(1), expectArg("pathname", "/tmp/netconfig")),
+						expectEvent(anyHost, "cp", testutils.CPUForTests, anyPID, 0, events.SecurityFileOpen, orPolNames("sfo-mw-pol-1"), orPolIDs(1), expectArg("pathname", "/tmp/passwd")),
+						expectEvent(anyHost, "cp", testutils.CPUForTests, anyPID, 0, events.MagicWrite, orPolNames("sfo-mw-pol-1"), orPolIDs(1), expectArg("pathname", "/tmp/passwd")),
+					},
+					[]string{},
+				),
+			},
+			useSyscaller: false,
+			coolDown:     0,
+			test:         ExpectAllInOrderSequentially,
+		},
+		{
+			name: "comm: event: data: trace event with pathname exceeding 255 characters",
+			policyFiles: []testutils.PolicyFileWithID{
+				{
+					Id: 1,
+					PolicyFile: v1beta1.PolicyFile{
+						Metadata: v1beta1.Metadata{
+							Name: "sfo-pol-1",
+						},
+						Spec: k8s.PolicySpec{
+							Scope: []string{
+								"comm=more",
+							},
+							DefaultActions: []string{"log"},
+							Rules: []k8s.Rule{
+								{
+									Event: "security_file_open",
+									Filters: []string{
+										"data.pathname=/tmp/00000100000000*",
+									},
+								},
+							},
+						},
+					},
+				},
+				{
+					Id: 2,
+					PolicyFile: v1beta1.PolicyFile{
+						Metadata: v1beta1.Metadata{
+							Name: "sfo-pol-2",
+						},
+						Spec: k8s.PolicySpec{
+							Scope: []string{
+								"comm=more",
+							},
+							DefaultActions: []string{"log"},
+							Rules: []k8s.Rule{
+								{
+									Event: "security_file_open",
+									Filters: []string{
+										"data.pathname=*2500056789",
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			cmdEvents: []cmdEvents{
+				newCmdEvents(
+					"bash -c 'touch /tmp/00000100000000200000000030000000004000000000500000000060000000007000000000800000000090000000001"+
+						"0000000001100000000120000000013000000001400000000150000000016000000001700000000180000000019000000002000000000210000000"+
+						"02200000000230000000024000000002500056789; more /tmp/00000100000000200000000030000000004000000000500000000060000000007"+
+						"0000000008000000000900000000010000000001100000000120000000013000000001400000000150000000016000000001700000000180000000"+
+						"01900000000200000000021000000002200000000230000000024000000002500056789'",
+					0,
+					1*time.Second,
+					[]trace.Event{
+						expectEvent(anyHost, "more", testutils.CPUForTests, anyPID, 0, events.SecurityFileOpen, orPolNames("sfo-pol-1", "sfo-pol-2"), orPolIDs(1, 2),
+							expectArg("pathname", "/tmp/00000100000000200000000030000000004000000000500000000060000000007000000000800000000090000000001"+
+								"0000000001100000000120000000013000000001400000000150000000016000000001700000000180000000019000000002000000000210000000"+
+								"02200000000230000000024000000002500056789")),
```

Comment:
These file names will be hard to handle in future changes. Is it possible to use alpha chars instead?